src/main/java/com/endava/soa_5/constant/UsersCredentials.java

```diff
@@ -0,0 +1,21 @@
+package com.endava.soa_5.constant;
+
+import lombok.Getter;
+
+@Getter
+public enum UsersCredentials {
+    EDUARD("gafax88355@fironia.com", "eduard_soa5"),
```

Comment:
credential is better to be written in properties files

Reply:
done